ui/src/app/base/components/FormCard/FormCard.scss

```diff
@@ -4,6 +4,12 @@
   margin-top: 0.5rem;
 }
 
+.form-card__title {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
Since "Editing" is always displayed there's not much room for the actual name, so I think this looks better as:
``` scss
.form-card__title {
  word-break: break-word;
}
```

Reply:
Good suggestion.